packages/htmlbars-compiler/tests/html-compiler-test.js

```diff
@@ -1012,6 +1013,8 @@ test("A helpful error message is provided for mismatched start/end tags", functi
   }, /Closing tag `div` \(on line 5\) did not match last open tag `p` \(on line 2\)\./);
 });
 
+if (innerHTMLHandlesNewlines) {
```

Comment:
Sure, an inline comment would be helpful so we don't forget.

If you make that change, please merge! :-)